contracts/modules/cover/Cover.sol

```diff
@@ -470,7 +504,7 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
         * burnAmount / segment.amount
         * GLOBAL_CAPACITY_DENOMINATOR / globalCapacityRatio;
 
-      stakingPool(i).burnStake(cover.productId, segment.start, segment.period, burnAmountInNXM);
+      stakingPool(i).burnStake(burnAmountInNXM);
```

Comment:
What's the intention here with burnStake?

It would still require productId i'd assume and time interval to know the tranches?

Reply:
Burning staked nxm affects all active tranches regardless of the cover buy time and period.